contracts/actions/checkers/MorphoBlueTargetRatioCheck.sol

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity =0.8.24;
+
+import { MorphoBlueHelper } from "../../actions/morpho-blue/helpers/MorphoBlueHelper.sol";
+import { MarketParams } from "../../interfaces/morpho-blue/IMorphoBlue.sol";
+import { ActionBase } from "../ActionBase.sol";
+
+/// @title Check if the current ratio for morpho position is within a certain range of the target ratio
+contract MorphoBlueTargetRatioCheck is ActionBase, MorphoBlueHelper {
```

Comment:
Would change name to MorphoBlueOpenRatioCheck, to keep it consistent with Aave action naming

Reply:
I would leave it as is and name all future checks in this manner because it better describes what it does. "AaveV3OpenRatioCheck" was not a great name since we ended up using it for b/r on price strategies, wouldn't change it now, though, but I would use this naming convention for future strategies instead